xml/art_sle_amazon_ecs.xml

```diff
@@ -105,13 +112,13 @@
       ECS specific policies and roles can be found in the Amazon documentation:
       <link xlink:href="http://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html">IAM Role</link>
     </para>
-    <para>
+    <note>
       Note that the policy given above is more permissive than the
```

Comment:
Wrap the text inside `<para>...</para>` as this isn't a valid DocBook construct.